pkg/controller/container-runtime-config/helpers.go

```diff
@@ -449,6 +456,9 @@ func updatePolicyJSON(data []byte, internalBlocked, internalAllowed []string, re
 			transportScopes[reg] = signature.PolicyRequirements{
 				signature.NewPRInsecureAcceptAnything(),
 			}
+			if r.FindAllStringIndex(reg, -1) == nil {
```

Comment:
(`MatchString` seems to be simpler and more efficient — this unnecessarily allocates an array to hold matches — but really the regexp is probably not worth it overall.)